src/components/AdvantagesList/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import Img from 'gatsby-image'
+
+const Wrap = styled.div`
+	margin-top: 42px;
```

Comment:
Try to use the outside-in CSS organization. We try to follow these guidelines in all our projects, so it's easier for everyone to read CSS. Please resolve this in all scss

https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685

Reply:
okay, will fix